src/routes/auth.routes.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import jwt from 'jsonwebtoken';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import bcrypt from 'bcrypt';
+import db from '../database/models';
+
+const router = express.Router();
+
+// eslint-disable-next-line operator-linebreak
+const JWT_SECRET =
+  'goK!pusp6ThEdURUtRenOwUhAsWUCLheBazl!uJLPlS8EbreWLdrupIwabRAsiBu';
+
+router.post('/login', async (req, res) => {
+  const user = await db.user.findOne({ where: { email: req.body.email } });
+
+  if (!user) {
+    return res.status(401).send({ message: 'Incorrect email or password.' });
+  }
+
+  const isPasswordValid = bcrypt.compareSync(req.body.password, user.password);
+  if (!isPasswordValid) {
+    return res.status(401).send({ message: 'Incorrect email or password.' });
+  }
+
+  const token = jwt.sign({ email: user.email }, JWT_SECRET);
+
+  res.status(200).send({ id: user.id, email: user.email, token });
+});
+
+export default router;
```

Comment:
I think this file isn't necessary,
better you can remove it.

Reply:
done